javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,62 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === '' || studentName === undefined || studentName === null) {
+        return console.log(`Please provide a valid student name`);
+    }
+    else if (!istudentExist(studentName)) {
+        if (class07Students.length < 6 || studentName.toUpperCase() === 'QUEEN') {
+            console.log(`${studentName} has been added successfully`);
+            class07Students.push(studentName);
+        }
+        else {
+            console.log(`Sorry the class is full to enroll for any more student`);
+        }
+    }
+    else
+        console.log(`${studentName} is already exists in the class`);
+}
+
+let istudentExist = studentName => class07Students.find(name => name.toLowerCase() === studentName.toLowerCase()
```

Comment:
You could use an easier way to check if a value exist in an array. By easier I mean less lines of codes.

Reply:
Yes I can do that with includes,as we already used include in the class so thought to use find() and arrow fucntion just for exposure.

Reply:
I understand but don't worry you will get plenty of time to practice higher order methode. Let's keep focusing on writing clean and clear code which is readable and doesn't take a lot of space.

Reply:
Sure will keep in mind, thank you.